### PR TITLE
fix: fix log format

### LINKED ISSUE #1270: Update tailcall log format
**Current format**
```
[2024-03-02T05:52:51Z INFO  tailcall::cli::file] File read: ./examples/jsonplaceholder.graphql ... ok
[2024-03-02T05:52:51Z INFO  tailcall::cli::tc] N + 1: 1
[2024-03-02T05:52:51Z INFO  tailcall::cli::server] 🚀 Tailcall launched at [0.0.0.0:8000] over HTTP/1.1
[2024-03-02T05:52:51Z INFO  tailcall::cli::server] 🌍 Playground: http://127.0.0.1:8000
[2024-03-02T05:52:52Z WARN  tailcall::cli::update_checker] A new release of tailcall is available: v0.35.0 ➜ v0.40.9
[2024-03-02T05:52:52Z WARN  tailcall::cli::update_checker] To upgrade, run: brew upgrade tailcall
[2024-03-02T05:52:52Z WARN  tailcall::cli::update_checker] https://github.com/tailcallhq/tailcall/releases/tag/v0.40.9
```

**Proposed Format**

```
[INFO] File read: ./examples/jsonplaceholder.graphql ... ok
[INFO] N + 1: 1
[INFO] 🚀 Tailcall launched at [0.0.0.0:8000] over HTTP/1.1
[INFO] 🌍 Playground: http://127.0.0.1:8000
[WARN] A new release of tailcall is available: v0.35.0 ➜ v0.40.9
[WARN] To upgrade, run: brew upgrade tailcall
[WARN] https://github.com/tailcallhq/tailcall/releases/tag/v0.40.9
```

### PR DESCRIPTION
**Pull request by 35C4n0r**
**Summary:**  
This PR tries to increase the readability of the log messages, by removing the timestamp and targets.

**Issue Reference(s):**  
Fixes #1270 
/claim #1270 

image

**Build & Testing:**

- [x] I ran `cargo test` successfully.
- [x] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

**Checklist:**

- [ ] I have added relevant unit & integration tests.
- [ ] I have updated the [documentation] accordingly.
- [x] I have performed a self-review of my code.
- [ ] PR follows the naming convention of `<type>(<optional scope>): <title>`

[documentation]: https://github.com/tailcallhq/tailcallhq.github.io/tree/develop/docs

### CODE REVIEW

File: src/cli/tc.rs [hunk 1]
```diff
@@ -188,5 +189,12 @@ fn logger_init() {
...
 
-    env_logger::Builder::from_env(env).init();
+    env_logger::Builder::from_env(env)
+        .format(|buf, record| {
+            // Custom formatting for log messages
+            let level = record.level();
+            let args = record.args();
+            writeln!(buf, "[{}] {}", level, args)
+        })
+        .init();
```
Comment (tusharmath) [inline src/cli/tc.rs:199]: color coding is getting lost.

Comment (35C4n0r) [reply src/cli/tc.rs:208]: @tusharmath Made the changes.

**Comment by 35C4n0r** [comment]
@tusharmath PR is ready to review

**Comment by tusharmath** [comment]
closing in favour of https://github.com/tailcallhq/tailcall/pull/1272

**Pull request closed without merging**